src/shared/timeseries/index.js

```diff
@@ -190,7 +190,8 @@ async function generateTimeseries(options = {}) {
   const lastDate = dates[dates.length - 1];
   let featureCollection;
   for (const date of dates) {
-    const data = await generate(date === today ? undefined : date, {
+    const data = await runCrawler({
```

Comment:
For a second, I thought it would be fun push all of the promises to an array and do an `await Promise.all(promisesArrray)`, but then I realized we'd probably run out of memory even faster :)

Reply:
Yes, that would probably break things haha. Let's start with keeping this process sequential.